### PR TITLE
props to theano/misc/pycuda_example.py

### PR DESCRIPTION
**Pull request by harlouci**
props.

### CODE REVIEW

File: theano/misc/pycuda_example.py [hunk 1]
```diff
@@ -279,6 +279,7 @@ def perform(self, node, inputs, out):
 class PycudaElemwiseSourceModuleMakeThunkOp(Op):
     nin = property(lambda self: self.scalar_op.nin)
     nout = property(lambda self: self.scalar_op.nout)
+    __props__ = ("scalar_op", "inplace_pattern", "name")
```
Comment (nouiz) [inline theano/misc/pycuda_example.py]: remove name from the props.

**Comment by nouiz** [comment]
i canceled travis on this one.

**Comment by nouiz** [comment]
@harlouci can you finish this one?

**Comment by harlouci** [comment]
I removed 'name' from the props.

There is a problem in 'theano.misc.pycuda_init.py' with the value of variable 'pycuda_available' that set to False raises en exception at line 38 in 'theano/misc/pycuda_example.py' when I run theano-nose.

**Comment by nouiz** [comment]
you didn't push the new version. It is normal that you can't run those example, as I think you don't have pycuda installed.

**Comment by harlouci** [comment]
Sorry, it's seems indeed that I didn't push my last commit... Here it is.

**Pull request merged**